Description: tell the build script how to link to libpng
Origin: vendor
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-12-26

--- a/build/rrd.sh
+++ b/build/rrd.sh
@@ -3,7 +3,7 @@
 	RRDDEF=""
 	RRDINC=""
 	RRDLIB=""
-	PNGLIB=""
+	PNGLIB="$(libpng12-config --ldflags)"
 	ZLIB=""
 	for DIR in /opt/rrdtool* /usr/local/rrdtool* /usr/local /usr /usr/pkg /opt/csw /opt/sfw /usr/sfw
 	do
